app/src/main/java/io/github/polysmee/AppointmentReminderNotificationPublisher.java

```diff
@@ -0,0 +1,78 @@
+package io.github.polysmee;
+
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.provider.Settings;
+
+import androidx.core.app.NotificationCompat;
+import androidx.core.app.NotificationManagerCompat;
+
+import java.util.concurrent.TimeUnit;
+
+
+/**
+ * greatly inspired by https://developer.android.com/training/notify-user/build-notification#java
+ * It is the broadcast receiver class that will receive broadcasts at certain times (specified in
+ * in the values resources, in appointmentReminderNotification.xml) before appointments, and will create
+ * a notification at each broadcast received to remind the user that he/she has a appointment coming soon
+ *
+ * @author Thomas Bienaimé
+ **/
```

Comment:
Not needed cf https://stackoverflow.com/questions/17269843/javadoc-author-tag-good-practices

Reply:
Really good point, thanks. I wrote it in the first place so that if there is a question he would know whom to ask, but with a version control system it is possible and much more easy!